### ISSUE TITLE
When my local app version and PlayStore version are the same after that show this pop-up.

### ISSUE THREAD

**Issue by HappySinha** [timtraversy/new_version#62]
I don't know why I'm facing this issue.  When my local app version and PlayStore version are the same after that show this pop-up.

I'm using very simple code and default code

      @override
       void initState() {
          super.initState();
          ...
          checkVersion();
        }

      void checkVersion() async {
        final newVersion = NewVersion(
          androidId: "banda.app",
        );

      final status = await newVersion.getVersionStatus();
      newVersion.showUpdateDialog(
        versionStatus: status,
        context: context,
      );
    }

This is a Screenshot:- https://user-images.githubusercontent.com/53011602/133896317-4ba20fa4-0e10-4c05-b02f-193073fe070b.jpg

This code work on our project.

      @override
       void initState() {
          super.initState();
          ...
          checkVersion();
        }
    void checkVersion() async {
      final newVersion = NewVersion(
        androidId: "banda.app",
      );
  
      final status = await newVersion.getVersionStatus();
    
    // I'm checking manual add If condition
     if (status.localVersion != status.storeVersion) {
        newVersion.showUpdateDialog(
          versionStatus: status,
          context: context,
        );
      }
    }

**Comment by timtraversy** [comment]
Hello - do you still need assistanc here?

**Comment by HappySinha** [comment]
Yes I need assistance. Can you provide me please

**Comment by HappySinha** [comment]
> Hello - do you still need assistance here?

Yes I need assistance. Can you provide me please

**Comment by timtraversy** [comment]
Ah ok - you closed this issue so I thought it might be resolved.

What values are `localVersion` and `storeVersion` being set as?

**Comment by HappySinha** [comment]
Yes I'm solved this is issue in my code. But you very need to resolve this issue.
I request you.
Please please please 🙏

**Comment by HappySinha** [comment]
Oh yes | Thank-you so much for helping me
